content-api/content-actors/src/main/scala/org/sunbird/util/ChannelConstants.scala

```diff
@@ -14,4 +14,7 @@ object ChannelConstants {
   val OBJECT_CATEGORY: String = "ObjectCategory"
   val categoryKeyList: List[String] = List(CONTENT_PRIMARY_CATEGORIES,COLLECTION_PRIMARY_CATEGORIES,ASSET_PRIMARY_CATEGORIES)
   val ERR_VALIDATING_PRIMARY_CATEGORY: String = "ERR_VALIDATING_PRIMARY_CATEGORY"
+  val CONTENT_ADDITIONAL_CATEGORIES: String = "contentAdditionalCategories"
```

Comment:
CONTENT_ADDITIONAL_CATEGORIES, COLLECTION_ADDITIONAL_CATEGORIES and ASSET_ADDITIONAL_CATEGORIES should be added into 
val categoryKeyList: List[String] = List(CONTENT_PRIMARY_CATEGORIES,COLLECTION_PRIMARY_CATEGORIES,ASSET_PRIMARY_CATEGORIES)

Otherwise these fields in metadata will not be validated with master category in `validateObjectCategory` method.